consultation_analyser/consultations/models.py

```diff
@@ -273,6 +273,7 @@ class CandidateTheme(UUIDPrimaryKeyModel, TimeStampedModel):
     question = models.ForeignKey(Question, on_delete=models.CASCADE)
     name = models.TextField()
     description = models.TextField()
+    approximate_frequency = models.IntegerField(null=True)
```

Comment:
```suggestion
    approximate_frequency = models.IntegerField(null=True, blank=True)
```
this is useful for the admin